### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Create  table named owners.
- [x] Create a table named species.
- [x] Modify animals table.
- [x] Insert data in owners table.
- [x] Insert data in species table.
- [x] Create new queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -36,3 +38,14 @@ SELECT AVG(weight_kg) FROM animals;
...
+/* MILESTONE 3: query multiple tables */
+
+SELECT name, full_name FROM owners INNER JOIN animals ON owners.id = animals.owner_id WHERE owners.full_name = 'Melody Pond';
+SELECT species.name, animals.name FROM species INNER JOIN animals ON species.id = animals.species_id WHERE species.name = 'Pokemon';
+SELECT name, full_name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id;
+SELECT COUNT(animals.name) FROM species INNER JOIN animals ON species.id = animals.species_id WHERE species.name = 'Pokemon';
+SELECT COUNT(animals.name) FROM species INNER JOIN animals ON species.id = animals.species_id WHERE species.name = 'Digimon';
+SELECT name FROM animals INNER JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Jennifer Orwell' AND animals.species_id = 2;
+SELECT animals.name, full_name FROM owners INNER JOIN animals ON owners.id = animals.owner_id WHERE full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+SELECT COUNT(animals.name), owners.full_name FROM owners INNER JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name;
```
Comment (DammyShittu) [inline queries.sql:51]: - Great queries so far. 👍 However, the last query says; `Who owns the most animals?`.
After running your query, this 👇 is what I got;



To get the person who owns the most animals, I believe the query should be written this 👇 way;

```
SELECT owners.full_name, COUNT(owners.full_name) FROM animals  
LEFT JOIN owners   ON animals.species_id = owners.id  
GROUP BY owners.full_name  
ORDER BY COUNT(owners.full_name) DESC LIMIT 1;
```
😉

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @fabgrel10, 👋 

You have done an amazing job so far. 

Well done 👍 

There is a little issue that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

_Check the comment under the review._


# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (fabgrel10) [reply queries.sql]: Hello @DammyShittu ,

Thank you for pointing that specific query. I couldn't get the result I was looking for. After running your query I got this:

ksnip_20220323-134854

After a small change I got the expected result:

ksnip_20220323-135039

I corrected the query now, thank you for your feedback and assistance!

**Review by descholar-ceo — APPROVED** [review]
Hi @fabgrel10 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fabgrel10** [comment]
Thank you for the review @descholar-ceo!

**Pull request merged**